### DISCUSSION TITLE
Interested in two proposals for GSoc 2023

### DISCUSSION THREAD

**Discussion by AmineZouitine** [SeaQL/summer-of-code#15] [Q&A]
Hi,

My name is Amine Zouitine 👋 and I am a computer science student at Epita Paris in France . I saw SeaQL in the 2022 edition of Google Summer of code, and I really enjoyed your project.  I am now highly motivated to contribute to your project in the 2023 edition.

The proposals that interest me the most are:

- Expand the features of SeaORM - An async & dynamic ORM for Rust

- Expand the features of Seaography - A GraphQL framework for SeaORM

I have been working with Rust 🦀 for the past year and have some projects available on my GitHub [profile.](https://github.com/AmineZouitine) Additionally, I have read your CONTRIBUTING.md and am eager to start contributing to `sea-orm` as soon as possible.

Do you have any particular advice for maximizing my chances of being selected for this year's Google Summer of Code program with your team? 🤗

Best regards,
Amine Zouitine (Discord: Amine_Ztn)

**Comment by tyt2y3** [comment]
You certainly have a good GitHub profile! To make it better, add a readme and summarize your interest, contribution and aspirations, like (shamelessly) [I do](https://github.com/tyt2y3).

I think it's important to have one or two PRs made to show your understanding of the subject matter and our project designs. Plus, being able to write good design documents with original, sensible ideas is key.

It may not be directly relevant, but I attached a sample of the [StarfishQL Design Doc](https://github.com/SeaQL/summer-of-code/blob/main/2022/StarfishQL%20Design%20Doc.pdf). You can also look at past [discussions](https://github.com/SeaQL/summer-of-code/discussions).
